graphql_compiler/schema/sqlalchemy_schema.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+
+# Complete schema information sufficient to compile GraphQL queries to SQLAlchemy
+SQLAlchemyMetadata = namedtuple('SQLAlchemyMetadata', (
+    'schema',  # GraphQLSchema
+    'tables',  # dict mapping every graphql type in the schema to a sqlalchemy table
```

Comment:
nit: GraphQLObjectType

Reply:
It's a string. Clarified.

Reply:
Apologies. I meant to say that it does not map every graphql type in the schema to a sqlalchemy table, but instead every graphql **object**  type to a sqlalchemy table.

Reply:
Not everything is a GraphQL object, there are also interfaces and enums etc. to worry about. I think GraphQL type is fine here.

Reply:
Technically, the GraphQLScalarType class is a subclass of the GraphQLType class. Therefore, this comment currently implies that we intend to map every graphql scalar type in the schema to a sqlalchemy table.

Reply:
Yes. But `GraphQLType` and `GraphQL type` (which I wrote) aren't the same thing. By not pointing to a specific type by exact name, I think the latter is asking the user to exercise some caution and common sense. `GraphQLObjectType` is less precise here, and unless there's a great option we're all missing, I think we should go with the least-bad option and move on to the meat of the changes instead of spending more time on this.